### ISSUE TITLE
Issue installing install_github("hadley/l1tf") for R version 4.3.0

### ISSUE THREAD

**Issue by mikeabata** [hadley/l1tf#8]
Hi,
When attempting to install the l1tf package I received a number of error message that I don't get when installing on 4.2.x versions. The package ultimately fails to load. These errors include the following:

l1tf.c:131:14: error: too few arguments to function 'dpbtrf_'
  131 |     F77_CALL(dpbtrf)("L",&m,&itwo,DDTF,&ithree,&info
l1tf.c:187:22: error: too few arguments to function 'dpbtrs_'
  187 |             F77_CALL(dpbtrs)("L",&m,&itwo,&ione,DDTF,&ithree,tmp_m1,&m,&info);
l1tf.c:189:22: error: too few arguments to function 'dgbtrs_'
  189 |             F77_CALL(dgbtrs)("N",&m,&itwo,&itwo,&ione,DDTF,&iseven,IPIV,tmp_m1,&m,&info);
l1tf.c:235:18: error: too few arguments to function 'dpbsv_'
  235 |         F77_CALL(dpbsv)("L",&m,&itwo,&ione,S,&ithree,dz,&m,&info); /* dz=S\r */
l1tf.c:336:14: error: too few arguments to function 'dpbsv_'
  336 |     F77_CALL(dpbsv)("L",&m,&itwo,&ione,mat,&ithree,vec,&m,&info);
ERROR: compilation failed for package 'l1tf'

Any help on this would be greatly appreciated!

**Comment by KevinCPhelps** [comment]
I am also having this issue.

**Comment by bholtdwyer** [comment]
Ditto. Here's my readout:

```
> install_github("hadley/l1tf")
Downloading GitHub repo hadley/l1tf@HEAD
── R CMD build ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
✔  checking for file 'C:\Users\holtd\AppData\Local\Temp\RtmpsdDZhA\remotes36d83a291280\hadley-l1tf-975038c/DESCRIPTION' (871ms)
─  preparing 'l1tf':
✔  checking DESCRIPTION meta-information ... 
─  cleaning src
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building 'l1tf_0.0.0.9000.tar.gz'
   
Installing package into ‘C:/Users/holtd/AppData/Local/R/win-library/4.3’
(as ‘lib’ is unspecified)
Loading required package: plyr
Loading required package: pacman
Loading required package: rlang
Loading required package: dplyr

Attaching package: 'dplyr'

The following objects are masked from 'package:plyr':

    arrange, count, desc, failwith, id, mutate, rename, summarise,
    summarize

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Loading required package: conflicted
Loading required package: readxl
Loading required package: testit
Loading required package: data.table
Loading required package: stats
Loading required package: Hmisc
Loading required package: sf
Linking to GEOS 3.11.2, GDAL 3.6.2, PROJ 9.2.0; sf_use_s2() is TRUE
Loading required package: tidyverse
── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ forcats   1.0.0     ✔ readr     2.1.4
✔ ggplot2   3.4.2     ✔ stringr   1.5.0
✔ lubridate 1.9.2     ✔ tibble    3.2.1
✔ purrr     1.0.1     ✔ tidyr     1.3.0
Loading required package: raster
Loading required package: sp
Loading required package: gdata
Loading required package: viridis
Loading required package: viridisLite
Loading required package: foreign
Loading required package: haven
[conflicted] Will prefer base::intersect over any other package.
[conflicted] Will prefer dplyr::summarize over any other package.
[conflicted] Will prefer dplyr::filter over any other package.
[conflicted] Will prefer dplyr::select over any other package.
[conflicted] Will prefer dplyr::summarise over any other package.
[conflicted] Will prefer dplyr::arrange over any other package.
[conflicted] Will prefer dplyr::rename over any other package.
[conflicted] Will prefer dplyr::mutate over any other package.
[conflicted] Will prefer terra::extract over any other package.
[conflicted] Will prefer testit::assert over any other package.
[conflicted] Will prefer base::setdiff over any other package.
[1] "I'm a Windows PC."
During startup - Warning message:
The `size` argument of `element_line()` is deprecated as of ggplot2 3.4.0.
ℹ Please use the `linewidth` argument instead. 
* installing *source* package 'l1tf' ...
** using staged installation
** libs
using C compiler: 'gcc.exe (GCC) 12.2.0'
gcc  -I"C:/PROGRA~1/R/R-43~1.0/include" -DNDEBUG     -I"C:/rtools43/x86_64-w64-mingw32.static.posix/include"     -O2 -Wall  -mfpmath=sse -msse2 -mstackrealign  -c l1tf-r.c -o l1tf-r.o
gcc  -I"C:/PROGRA~1/R/R-43~1.0/include" -DNDEBUG     -I"C:/rtools43/x86_64-w64-mingw32.static.posix/include"     -O2 -Wall  -mfpmath=sse -msse2 -mstackrealign  -c l1tf.c -o l1tf.o
In file included from C:/PROGRA~1/R/R-43~1.0/include/R.h:78,
                 from l1tf.c:10:
l1tf.c: In function 'l1tf':
l1tf.c:131:14: error: too few arguments to function 'dpbtrf_'
  131 |     F77_CALL(dpbtrf)("L",&m,&itwo,DDTF,&ithree,&info);
      |              ^~~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:759:10: note: declared here
  759 | F77_NAME(dpbtrf)(const char* uplo, const int* n, const int* kd,
      |          ^~~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:759:1: note: in expansion of macro 'F77_NAME'
  759 | F77_NAME(dpbtrf)(const char* uplo, const int* n, const int* kd,
      | ^~~~~~~~
l1tf.c:187:22: error: too few arguments to function 'dpbtrs_'
  187 |             F77_CALL(dpbtrs)("L",&m,&itwo,&ione,DDTF,&ithree,tmp_m1,&m,&info);
      |                      ^~~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:765:10: note: declared here
  765 | F77_NAME(dpbtrs)(const char* uplo, const int* n,
      |          ^~~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:765:1: note: in expansion of macro 'F77_NAME'
  765 | F77_NAME(dpbtrs)(const char* uplo, const int* n,
      | ^~~~~~~~
l1tf.c:189:22: error: too few arguments to function 'dgbtrs_'
  189 |             F77_CALL(dgbtrs)("N",&m,&itwo,&itwo,&ione,DDTF,&iseven,IPIV,tmp_m1,&m,&info);
      |                      ^~~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:168:10: note: declared here
  168 | F77_NAME(dgbtrs)(const char* trans, const int* n,
      |          ^~~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:168:1: note: in expansion of macro 'F77_NAME'
  168 | F77_NAME(dgbtrs)(const char* trans, const int* n,
      | ^~~~~~~~
l1tf.c:235:18: error: too few arguments to function 'dpbsv_'
  235 |         F77_CALL(dpbsv)("L",&m,&itwo,&ione,S,&ithree,dz,&m,&info); /* dz=S\r */
      |                  ^~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:734:10: note: declared here
  734 | F77_NAME(dpbsv)(const char* uplo, const int* n,
      |          ^~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:734:1: note: in expansion of macro 'F77_NAME'
  734 | F77_NAME(dpbsv)(const char* uplo, const int* n,
      | ^~~~~~~~
l1tf.c: In function 'l1tf_lambdamax':
l1tf.c:336:14: error: too few arguments to function 'dpbsv_'
  336 |     F77_CALL(dpbsv)("L",&m,&itwo,&ione,mat,&ithree,vec,&m,&info);
      |              ^~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:734:10: note: declared here
  734 | F77_NAME(dpbsv)(const char* uplo, const int* n,
      |          ^~~~~
C:/PROGRA~1/R/R-43~1.0/include/R_ext/RS.h:77:25: note: in definition of macro 'F77_CALL'
   77 | # define F77_CALL(x)    x ## _
      |                         ^
C:/PROGRA~1/R/R-43~1.0/include/R_ext/Lapack.h:734:1: note: in expansion of macro 'F77_NAME'
  734 | F77_NAME(dpbsv)(const char* uplo, const int* n,
      | ^~~~~~~~
make: *** [C:/PROGRA~1/R/R-43~1.0/etc/x64/Makeconf:265: l1tf.o] Error 1
ERROR: compilation failed for package 'l1tf'
```

**Comment by Amber-SUSTECH** [comment]
Me too! Same problem

**Comment by yizhenxu** [comment]
any solutions? TT

**Comment by hadley** [comment]
Fixed by #9
